tools/genesis/genesis.go

```diff
@@ -128,18 +123,16 @@ func generateAllocations(
 			AddressStates: genesis.AddressStates{ConsortiumMember: isConsortiumMember, KYCVerified: isKycVerified},
 		}
 
-		if offerValueMinDuration != 0 && offerValueUnlockPeriodDuration != 0 {
-			depositOfferID, ok := offerValueToID[offerValueIndex]
-			if !ok {
-				fmt.Println("Skipping Row # ", al.RowNo, " Reason: No fitting allocation found for values -- index: ", offerValueIndex)
-				skippedRows++
-				continue
+		if hasOffer {
+			oid, _ := ids.FromString(offer.OfferID)
+			duration := offer.MinDuration
+			if offer.MinDuration <= al.DepositDuration && al.DepositDuration <= offer.MaxDuration {
```

Comment:
If that check fails it shouldn't be just ignored and put to the minimum value but throw an error message instead